modules/consul-cluster/variables.tf

```diff
@@ -195,3 +195,13 @@ variable "tags" {
   type        = "list"
   default     = []
 }
+
+variable "enable_iam_setup" {
+  description = "If false, the IAM policy and role setup will be disabled.  This is to allow the IAM to be setup elsewhere if required."
```

Comment:
A bit awkward to read the negative first. Perhaps, "If true, create the IAM Role, IAM Instance Profile, and IAM Policies. If false, these will not be created, and you can pass in your own IAM Instance Profile via var.iam_instance_profile_name".

Reply:
No problem i will replace.